Source/WebKit/WebProcess/WebPage/UnifiedTextReplacementController.h

```diff
@@ -97,6 +98,9 @@ class UnifiedTextReplacementController final {
 
     WeakPtr<WebPage> m_webPage;
 
+    typedef std::pair<WTF::UUID, WebCore::CharacterRange> TextIndicatorCharacterRange;
```

Comment:
can this be a HashMap?

Reply:
I avoided a hash map on purpose because of the empty problem.